pyVim/connect.py

```diff
@@ -804,6 +804,37 @@ def SmartConnect(protocol='https', host='localhost', port=443, user='root', pwd=
                   b64token=b64token,
                   mechanism=mechanism)
 
+def ConnectNoSSL(protocol='https', host='localhost', port=443, user='root', pwd='',
```

Comment:
Thanks @JLHasson 
Can we change the function name to SmartConnectNoSSL as it's a wrapper of SmartConnect? Do you plan to make the same change for Connect?